library/network/test/network_service_test.rb

```diff
@@ -23,6 +23,13 @@
         expect { Yast::NetworkService.is_netconfig }.not_to raise_error
       end
     end
+
+    describe "#EnableDisableNow" do
+      it "does not crash when current / cached service is nil" do
```

Comment:
Instead of not crashing what about describing the behavior...

```ruby
context "when a network service is running" do
  it "stops the current service"  do
  end
  it "disables the current service" do
  end
end

context "when a network service has been selected to be enabled" do
  it "enables the selected service" do
  end
end
....
```

Reply:
it is test which covers the issue from the bug report

Reply:
True, but the logic is not so big, so what about improving the test coverage?

Reply:
ok, will add something ;-)

Reply:
The responsibility of this module / class is well defined and isolated, so it does not need to be rewritten / re-factorized as much as other classes, but the test coverage is very low, then we should improve when we are able to, and I think it will not take so much time.

Reply:
yes, I agree and already promised that will add some tests ;-)